app/helpers/react_components/track/exercise_community_solutions_list.rb

```diff
@@ -11,13 +11,21 @@ def to_s
       def request
         {
           endpoint: Exercism::Routes.api_track_exercise_community_solutions_url(exercise.track, exercise),
-          query: params.slice(*AssembleExerciseCommunitySolutionsList.keys),
+          query: search_params,
           options: { initial_data: data }
         }
       end
 
       def data
-        AssembleExerciseCommunitySolutionsList.(exercise, params)
+        AssembleExerciseCommunitySolutionsList.(exercise, search_params)
+      end
+
+      memoize
+      def search_params
+        params.permit(*AssembleExerciseCommunitySolutionsList.keys).tap do |params|
+          # Set passed_head_tests to true by default
+          params[:passed_head_tests] = true if params[:passed_head_tests].blank?
+        end
```

Comment:
Feels like this needs to be dried up with the bit above I commented on.

Reply:
This is to have the passed head tests option be enabled by default in React.